pkg/controller/syncmachineset/syncmachineset_controller.go

```diff
@@ -288,3 +280,10 @@ func getGCPClientCredentials(kubeClient client.Client, cd *hivev1.ClusterDeploym
 
 	return gcpCreds, nil
 }
+
+func isPlatformSupported(cd *hivev1.ClusterDeployment) bool {
+	if cd.Spec.Platform.GCP != nil {
```

Comment:
I know that there is an expectation of supporting more platforms, but for now...
```
return cd.Spec.Platform.GCP != nil
```